_includes/menu.html

```diff
@@ -1,6 +1,6 @@
 <nav class="fixed-top full-width">
     <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large">
-        <a href="#home"><img src="{{ "img/logo.svg" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
+        <a href="/"><img src="{{ "img/logo.svg" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
```

Comment:
Should this be `/#home`? Haven't tested.

Reply:
Hm, guess it depends if you think this should always go to the top of the page. As it is now, this logo just goes to the home page, and if you're already there it does nothing. I don't feel strongly about it either way.

Reply:
Actually, I guess it refreshes the page. So I'd prefer keeping it "/" so the URL is clean.

Reply:
sgtm